src/Yahoo/Provider.php

```diff
@@ -39,13 +42,13 @@ protected function getTokenUrl()
      */
     protected function getUserByToken($token)
     {
-        $response = $this->getHttpClient()->get('https://social.yahooapis.com/v1/user/'.$this->xoauth_yahoo_guid.'/profile?format=json', [
```

Comment:
Since you don't use `xoauth_yahoo_guid` class property anymore, can you remove it?

Reply:
done